config/tests/dns/dns.yaml

```diff
@@ -0,0 +1,8 @@
+- op: add
+  path: "/spec/version"
+  value: 7.1.0
```

Comment:
We might want to make a note here that this version is yet not released and doesn't exist.

Reply:
I've added a note on that to the kustomization file.

Reply:
Thanks!